content/tutorials/queue-amqp-neutrino-profanity.textile

```diff
@@ -1,5 +1,5 @@
 ---
-ably_product: reactor
+ably_product: integrations
```

Comment:
I think this may need to stay as `reactor` for all tutorials, see [this note](https://github.com/ably/docs/tree/main/config/tutorials#deprecated-attributes). I'm unsure if they're *not* used because they're under "deprecated", or they're deprecated but still used for only the listed use case.